### DISCUSSION TITLE
XFORMERS error

### DISCUSSION THREAD

**Discussion by McQotoCovayashi** [TheLastBen/fast-stable-diffusion#2914] [General]
I am getting an XFORMERS error. What should I do? I am using Google colb.
-----
WARNING[XFORMERS]: xFormers can't load C++/CUDA extensions. xFormers was built for:
    PyTorch 2.3.0+cu121 with CUDA 1201 (you have 2.4.0+cu121)
    Python  3.10.14 (you have 3.10.12)
  Please reinstall xformers (see https://github.com/facebookresearch/xformers#installing-xformers)
  Memory-efficient attention, SwiGLU, sparse and more won't be available.
  Set XFORMERS_MORE_DETAILS=1 for more details

**Comment by Brandyfan** [comment]
I had the same issue and tried this suggestion- https://github.com/TheLastBen/fast-stable-diffusion/issues/2913#issuecomment-2311068983

That appeared to fix it as it's working now for me.

**Reply by McQotoCovayashi** [reply]
Oh! Thanks! I wonder which cell will I try in...?

**Reply by McQotoCovayashi** [reply]
I was able to fix it! Thank you!!

**Reply by Brandyfan** [reply]
You're welcome! :)
